src/guides/v2.3/javascript-dev-guide/widgets/widget-row-builder.md

```diff
@@ -54,6 +54,50 @@ The following example shows a PHTML file using the script:
 </script>
 ```
 
+The following example shows a PHTML file using the script with some data:
+
+```html
+<script>
+    require([
+        'jquery',
+        'rowBuilder'
+    ], function ($) {
+        'use strict';
+
+        var data = {
+            formData: [
+                ["field1-name1", "field2-name1", "field3-name1"],
```

Comment:
Also it's good to see for the readers:
- sample of HTML code for `row-container`
- sample of HTML code for `row-template`

@DashukVolodymyr , could you please provide them?

Thank you!

Reply:
@atwix-integrations 
I added HTML code for `row-container` and `row-template`.
Could you please proceed with the code review?

Thank you!

Reply:
Looks ok to me. ✅ 

Thank you!